### ISSUE TITLE
Incorrect rewriting of dict(test=[])

### ISSUE THREAD

**Issue by wolfgangl** [Instagram/Fixit#125]
Example interaction: 
```
echo "dict(test=[])" > test.py
python -m  fixit.cli.apply_fix test.py 
Scanning 1 files
test.py

Encountered exception <class 'Exception'> for the following paths:
test.py
Running `pyre start` may solve the issue.
test.py:1:1 [applied fix]
    RewriteToLiteralRule: It's unnecessary to use a list or tuple within a call to dict sincethere is
    literal syntax for this type
Encountered exception <class 'Exception'> for the following paths:
test.py
test.py
test.py
Running `pyre start` may solve the issue.
All done! ✨ 🍰 ✨
1 file left unchanged.

Found 1 reports in 1 files in 0.33 seconds.
(py3_venv) wolf@ldp-wolf:~/ros_ws_py3/src/real_world$ cat test.py
{}
(py3_venv) wolf@ldp-wolf:~/ros_ws_py3/src/real_world$ 
```

RewriteToLiteralRule is losing a field in the dictionary.

**Comment by jimmylai** [comment]
Nice catch!
The current implementation didn't consider the keyward argument carefully for the case like the example.
https://github.com/Instagram/Fixit/blob/f189e053d0b52f63231de9146bb6ca51984210a0/fixit/rules/rewrite_to_literal.py#L72-L76

A quick mitigation to avoid autofix the example case is to update the matcher pattern to require the `Arg` to to have no keyword name.
https://libcst.readthedocs.io/en/latest/nodes.html#libcst.Arg

The case `dict(test=[], other=1)` should be autofixed as `{"test": [], "other": 1}`. We can add some new logic handle this separately since it require convert the parameter name as SimpleString and it's specific to `dict()`.

Feel free to submit PRs to mitigate it and/or fix the autofixer.

**Comment by amyreese** [comment]
Hey folks, we just released Fixit 2.0, which includes major changes to the core frameworks, linting engine, API, and CLI. In light of this new release, this issue is likely to be resolved or no longer relevant. Please take a look at the new [user guide](https://fixit.readthedocs.io/en/latest/guide.html#) or the [upgrade guide](https://fixit.readthedocs.io/en/latest/upgrade.html) for more information on how to migrate your projects to the new version.

If you believe this issue is still relevant or still occurring in the latest version, please feel free to open a new issue, or reply here with more details.

Thank you!
